src/ceph-volume/ceph_volume/devices/raw/list.py

```diff
@@ -18,6 +19,28 @@ def direct_report(devices):
     _list = List([])
     return _list.generate(devices)
 
+def _get_bluestore_info(dev):
+    out, err, ret = process.call([
+        'ceph-bluestore-tool', 'show-label',
+        '--dev', dev], verbose_on_failure=False)
+    if ret:
+        # ceph-bluestore-tool returns failure if device is not bluestore OSD
+        logger.debug('failed to get BlueStore info on device %s' % dev)
+        return None
+    oj = json.loads(''.join(out))
+    if dev not in oj:
+        return None
+    if oj[dev]['description'] != 'main':
+        # ignore non-main devices, for now
+        return None
```

Comment:
log debug here too?

Reply:
I opted to log info, but I can change it if desired.